### PR TITLE
Fqn javadocs

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**
This PR contains new settings for gradle build files to enable FQN Javadoc Checker plugin. And few changes of links in javadocs that were found during smoke run.

### CODE REVIEW

**Comment by alexander-aleksandrov** [comment]
@dmytro-grankin PTAL

**Comment by dmytro-grankin** [comment]
@alexander-aleksandrov almost LGTM. Please address my comment.

File: client/build.gradle [hunk 1]
```diff
@@ -32,3 +32,5 @@ apply from: generateDescriptorSetScript
 apply from: testArtifactsScript
 
 apply plugin: spineProtobufPluginId
+
+apply plugin: spineJavadocFqnCheckerId
```
Comment (dmytro-grankin) [inline client/build.gradle]: Let's apply the plugin in `subprojects` section in the root project instead of applying it in many files.

Comment (alexander-aleksandrov) [inline client/build.gradle]: Done

**Pull request merged**